### ISSUE TITLE
FR: Progressbar ("highlight") color from hint

### ISSUE THREAD

**Issue by gustaphe** [dunst-project/dunst#861]
It would be nice if I could set up i3-volume to use the new progress bar behaviour, but then I'd want the progress bar to take different colors depending on the sent value, or a color sent with the notify-send.

```
$ notify-send -h int:value:100 -h string:highlight:"#ff0000" Volume
```
or similar (`hlcolor` to match the others maybe). Is there currently a way to do this? Can there be?

### Installation info
- Version: `Dunst - A customizable and lightweight notification-daemon 1.6.1 (2021-02-21)`
- Install type: `pacman package community/dunst`
- Distro and version: `Linux archlinux 5.11.16-arch1-1`

**Comment by fwSmit** [comment]
All the colors are set up as rules, so you can modify it with the rules. You can't match rules based on the progress value, but I can think of a hacky way to do this with hints, but we can easily support this dbus value. I guess I forgot to implement it last time. I've implemented it now in #862. This is based on my dev branch, so that's why there's so many commits, but only the last 2 commits are related to this issue.

**Comment by pc00per** [comment]
Implemented as of #862 but where's the solution ?

**Comment by vulpes-vulpeos** [comment]
For everyone who doesn't want to look into #862 code, solution is to add -h string:hlcolor:#ffffff
